### PR TITLE
Fixed the security white-paper link in cloud-native-security.md

### PR DESCRIPTION
**Pull request by vaibhav2107**
### Issue

Closes: #47650

### CODE REVIEW

File: content/en/docs/concepts/security/cloud-native-security.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ There are localized versions available of this whitepaper; if you can link to on
 when localizing, that's even better.
 {{< /comment >}}
 
-The CNCF [white paper](https://github.com/cncf/tag-security/tree/main/security-whitepaper)
+The CNCF [white paper](https://github.com/cncf/tag-security/tree/main/community/resources/security-whitepaper)
```
Comment (sftim) [inline content/en/docs/concepts/security/cloud-native-security.md:27]: I think we should link to https://github.com/cncf/tag-security/blob/main/community/resources/security-whitepaper/v2/CNCF_cloud-native-security-whitepaper-May2022-v2.pdf (see https://github.com/cncf/tag-security/tree/main/community/resources/security-whitepaper)

**Review by Okabe-Junya — COMMENTED** [review]
Great work, @vaibhav2107!

Please fix L207 too

https://github.com/kubernetes/website/blob/493ef1ba1cc078e77648c300819e16963e464397/content/en/docs/concepts/security/cloud-native-security.md?plain=1#L207

File: content/en/docs/concepts/security/cloud-native-security.md (hunk 1 above)
Comment (vaibhav2107) [reply content/en/docs/concepts/security/cloud-native-security.md:27]: @sftim Thanks for the suggestion. I will update the referenced link.

**Comment by vaibhav2107** [comment]
Thanks @Okabe-Junya for the review, I will update this

**Comment by vaibhav2107** [comment]
@sftim @Okabe-Junya I have updated the PR with your suggestions. Please have a look now

**Review by Okabe-Junya — COMMENTED** [review]
LGTM

**Review by dipesh-rawat — COMMENTED** [review]
Thanks!

/lgtm

[Preview Page](https://deploy-preview-47651--kubernetes-io-main-staging.netlify.app/docs/concepts/security/cloud-native-security/)

**Review by windsonsea — COMMENTED** [review]
/lgtm

**Pull request merged**